tests/helpers/mock-ui.js

```diff
@@ -25,6 +32,7 @@ MockUI.prototype.constructor = MockUI;
 MockUI.prototype.clear = function(){
   this.output = '';
   this.errors = '';
+  this.errorLog = [];
```

Comment:
I think I did have it storing and clearing those at one point, but it doesn't store the value defined in options now, so there's no way to reference it. I should probably rename the options to something less confusing. It's expecting an array for that value so it can push the messages to it.